yogstation/code/datums/components/fishingbonus.dm

```diff
@@ -4,22 +4,36 @@
 	var/mob/living/carbon/wearer
 
 /datum/component/fishingbonus/Initialize(fishing_bonus = 0)
-	if(!isclothing(parent))
+	if(!ismovable(parent))
 		return COMPONENT_INCOMPATIBLE
 	src.fishing_bonus = fishing_bonus
 	RegisterSignal(parent, COMSIG_ITEM_EQUIPPED, .proc/OnEquip)
```

Comment:
only register these signals to clothing

Reply:
No that wouldn't work on the /obj/vehicle then

Reply:
Notice the ones I am quoting, only move the equip ones to clothing.

Reply:
ok